API/api/services/auth_service.py

```
@@ -0,0 +1,29 @@
+# api/services/auth_service.py
+
+from api.repositories.auth_repository import AuthRepository
+from api.utils.jwt_utils import JwtUtils
+from rest_framework.exceptions import ValidationError, AuthenticationFailed
+
+
+class AuthService:
+    def __init__(self):
```

Ruff (0.15.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)